pytransform3d/transform_manager/_temporal_transform_manager.py

```diff
@@ -104,9 +103,11 @@ def check_transforms(self):
         return self
 
     def _interpolate_pq_using_sclerp(self, query_time):
+        if isinstance(query_time,float) or isinstance(query_time,float):
```

Comment:
We have to adapt typings [here](https://github.com/dfki-ric/pytransform3d/blob/97909fd124bed1dc560772c4bb08eca9efec958b/pytransform3d/transform_manager/_temporal_transform_manager.pyi#L78) as well since we allow both floats and arrays

Reply:
Also this line does not really make sense, right? Checks the same thing twice. It is also not very general. What if an int is passed? Or float32?